src/components/LoginCard/index.jsx

```diff
@@ -1,45 +1,49 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { Link, useHistory } from 'react-router-dom';
 
-import { Button, Card, Form, Input, Space, Typography } from 'antd';
+import { Button, Card, Form, Input, Modal, Space, Typography } from 'antd';
 
 import { GoogleCircleFilled, LockOutlined, UserOutlined } from '@ant-design/icons';
 
-import './LoginCard.scss';
-import { useAuth } from 'services/AuthProvider';
+import { useAuth } from 'hooks/Hooks';
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports